crates/pallets/starknet/runtime_api/src/lib.rs

```diff
@@ -71,7 +71,7 @@ sp_api::decl_runtime_apis! {
         /// client to operate seamlessly while abstracting the extrinsic complexity.
         fn extrinsic_filter(xts: Vec<<Block as BlockT>::Extrinsic>) -> Vec<Transaction>;
         /// Re-execute a block and return the TransactionExecutionInfos of every transaction in it, in the same order
```

Comment:
Change the doc and explain each argument better, the logic is becoming a bit complex